### DISCUSSION TITLE
Hey, does msquic support SRP feature in its tls implementation ? Thx.

### DISCUSSION THREAD

**Discussion by Kiddinglife** [microsoft/msquic#1919] [Q&A] [answered]
Hey, does msquic support SRP feature in its tls implementation ? Thx.

**Comment by thhous-msft** [comment, marked as answer]
SRP is not compatible with TLS 1.3, so no, and its not possible for any QUIC implementation to do so either, as TLS 1.3 is a requirement for QUIC.

**Reply by Kiddinglife** [reply]
Yea, Thx. I now using aws congnito that returns jwt token to the successfully authed client instead of cert. So look like I have to hack tls 1.3 auth flow to hook up the jwt validation logics via jwks.
